enforcer-rules/src/main/java/org/apache/maven/enforcer/rules/version/RequireJavaVersion.java

```diff
@@ -145,9 +145,9 @@ private void setCustomMessageIfNoneConfigured(ArtifactVersion detectedJdkVersion
                 getLog().debug("Could not parse allowed version range " + allowedVersionRange + " " + e.getMessage());
                 version = allowedVersionRange;
             }
-            String message = String.format(
-                    "Detected JDK version %s (JAVA_HOME=%s) is not in the allowed range %s.",
-                    detectedJdkVersion, SystemUtils.JAVA_HOME, version);
+            String message = "Detected JDK " + SystemUtils.JAVA_HOME
```

Comment:
single quote the value?

Reply:
I prefer not to. In my experience quote marks just get in the way of copy and paste, and I don't think there's any ambiguity here if we don't quote.